dot/sync/chain_sync.go

```diff
@@ -392,13 +404,16 @@ func (cs *chainSync) sync() {
 				continue
 			}
 
-			logger.Debug("worker error", "error", res.err.err)
+			logger.Debug("worker error", "worker ID", res.id, "error", res.err.err)
 
 			// handle errors. in the case that a peer did not respond to us in time,
 			// temporarily add them to the ignore list.
 			switch {
 			case errors.Is(res.err.err, context.Canceled):
 				return
+			case errors.Is(res.err.err, errNoPeers):
+				logger.Debug("not able to sync with any peer!", "worker ID", res.id)
+				continue
 			case errors.Is(res.err.err, context.DeadlineExceeded):
 				cs.ignorePeer(res.err.who)
 			case strings.Contains(res.err.err.Error(), "dial backoff"):
```

Comment:
Nit: Can we remove the empty default case here?

Reply:
yeah probably. I kinda like it for explicit-ness but can remove :p